src/modules/recommendations.js

```diff
@@ -120,6 +129,7 @@ class Recommendations {
    * @param {string} podId - Pod identifier
    * @param {object} [parameters] - Additional parameters to refine results
    * @param {string|array} [parameters.itemIds] - Item ID(s) to retrieve recommendations for (strategy specific)
+   * @param {string|array} [parameters.variationId] - Variation ID to retrieve recommendations for (strategy specific)
```

Comment:
I believe the type for this should just be `string`? I saw the RFC stating they'd want to prohibit multiple variation ids too.